homework/tests/example.spec.js

```diff
@@ -17,15 +17,115 @@ const data = [
 
 data.forEach(version => {
   test.describe(version + ': Add', () => {
-    test('Concatenating 2 and 3 results in 23', async ({ page }) => {
+    test('Adding 3,3 and 4 results in 7', async ({ page }) => {
       await page.goto('https://testsheepnz.github.io/BasicCalculator');
       await page.selectOption('#selectBuild', { label: version});
-      await page.locator('#number1Field').type('2');
-      await page.locator('#number2Field').type('3');
+      await page.locator('#number1Field').type('3,3');
```

Comment:
Use dot "." instead of comma ",", as comma is supported as string value instead of number value. You have caught an error "Number 1 is not a number" but your purpose was to check if if correct value is returned ".toHaveValue('7')" and not if error message is returned. If you want to check if error message is returned when incorrect values are provided to the input fields, separate test should be included.